src/components/Papers/PapersList.jsx

```diff
@@ -1,26 +1,46 @@
 /* global cozy */
-import React, { useMemo } from 'react'
-import get from 'lodash/get'
+import React, { useState, useMemo, useEffect } from 'react'
 
+import { useClient } from 'cozy-client'
+import { fetchCurrentUser } from 'src/utils/fetchCurrentUser'
 import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
 import ListSubheader from 'cozy-ui/transpiled/react/MuiCozyTheme/ListSubheader'
-import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import { Icon } from 'cozy-ui/transpiled/react'
 import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import Left from 'cozy-ui/transpiled/react/Icons/Left'
 
 import { getPapersByLabel } from 'src/utils/queries'
 import { useQuery } from 'src/components/Hooks/useQuery'
+import { useScannerI18n } from 'src/components/Hooks/useScannerI18n'
 import PaperLine from 'src/components/Papers/PaperLine'
 
 const PapersList = ({ history, match }) => {
+  const client = useClient()
+  const scannerT = useScannerI18n()
+  const [subheaderLabel, setSubheaderLabel] = useState(null)
   const { BarLeft } = cozy.bar
   const currentFileCategory = useMemo(
-    () => get(match, 'params.fileCategory', null),
+    () => match?.params?.fileCategory || null,
     [match]
   )
-  const { t } = useI18n()
   const { data: allPapers } = useQuery(getPapersByLabel(currentFileCategory))
+  const categoryLabel = scannerT(`items.${currentFileCategory}`)
+
+  useEffect(() => {
+    let isMounted = true
+    ;(async () => {
+      try {
+        const user = await fetchCurrentUser(client)
+        isMounted && setSubheaderLabel(user ? user.fullname : categoryLabel)
```

Comment:
Is it possible for `user.fullname` to be unset?

Reply:
Non, en passant par la méthode `findMyself`, on s'assure d'avoir un nom